scipy/stats/_stats_py.py

```diff
@@ -1463,9 +1484,10 @@ def skewtest(a, axis=0, nan_policy='propagate', alternative='two-sided'):
     pvalue : float
         The p-value for the hypothesis test.
 
-    Notes
-    -----
-    The sample size must be at least 8.
+    Raises
+    ------
+    ValueError
+        If shape of `a` along the given `axis` is less than 8.
```

Comment:
This is an example of where it is helpful information I think, because this is specific to `skewtest` and the error type isn't completely obvious based on the "less than 8" info in `Notes`.

Reply:
If we document this, can we remove the error in the future? I can see the document instead becoming ""... is less than ___, a Monte Carlo version of the test is performed.". Also, When there is `nan_policy='omit'`, it seems undesirable to raise an error when just one slice has fewer than the required number of elements. Many other stats functions just document these requirements and return NaN for the affected element when there is a problem.

Reply:
I don't see a problem with removing the error in the future.

And agreed to `'omit'`: returning `nan` seems reasonable. Going from error to non-error (`nan` or whatever) should always be fine, the reverse is typically not fine.